recipes/or-tools/build.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+cmake ${CMAKE_ARGS} -DCMAKE_INSTALL_PREFIX=$PREFIX \
```

Comment:
Please note that or-tools need c++17 and absl must use the SAME language dialect
-> verify you have this in the absl recipe https://github.com/Homebrew/homebrew-core/blob/master/Formula/abseil.rb#L24 (maybe with a comment to this recipe e.g. need absl/abseil recipe to use C++17 to build or-tools)

Reply:
EDIT: you should add it here: https://github.com/conda-forge/abseil-cpp-feedstock/blob/master/recipe/build.sh

note: I don't use conda so feel free to correct me ;)

Reply:
Yeah, that's probably why it was excluded the last time. Problem is that if I change something there it might break something in the overall conda-forge environment so let me see what I can do. 

> note: I don't use conda so feel free to correct me ;)

I normally build only run-of-the-mill python packages so any help is _much_ appreciated...!

Reply:
>here it works because they create the build dir "build" cd to it call cmake .. (aka to the parent directory) then call the build system (ninja).

That was the previous approach here but it failed. I'll move everything to cmake and the commands you suggested and then we should have something more concrete.

Reply:
> EDIT: you should add it here: https://github.com/conda-forge/abseil-cpp-feedstock/blob/master/recipe/build.sh

Please do add it there, we should tackle https://github.com/conda-forge/clang-compiler-activation-feedstock/issues/17 instead.

Reply:
> Please do add it there
I suspect you mean don't? 

I have to admit that that would be so far above my paygrade that it probably makes no sense if I start it 😄. Does that mean that we should build abseil here or wait until it is changed there? From how I understand the issue, Linux already uses c++17 so should we maybe restrict ourselves to that for now?

Reply:
> I suspect you mean don't here?

Yes 😂 

Start with Linux, we can add osx later once Linux is passing.

Reply:
Ok, will skip it :smile:. Maybe it's ready when I finally figure out how to build cmake recipes... 
Is there a flag or dependency I can set somewhere in case conda-forge ever updates to c++20 and or-tools is still at 17?